System/Console/ANSI/Windows/Emulator/Codes.hs

```diff
@@ -0,0 +1,83 @@
+{-# OPTIONS_HADDOCK hide #-}
+
+module System.Console.ANSI.Windows.Emulator.Codes
+    (
+      -- * Cursor movement by character
+      cursorUpCode, cursorDownCode, cursorForwardCode, cursorBackwardCode
+
+      -- * Cursor movement by line
+    , cursorUpLineCode, cursorDownLineCode
+
+      -- * Directly changing cursor position
+    , setCursorColumnCode, setCursorPositionCode
+
+      -- * Clearing parts of the screen
+    , clearFromCursorToScreenEndCode, clearFromCursorToScreenBeginningCode
+    , clearScreenCode, clearFromCursorToLineEndCode
+    , clearFromCursorToLineBeginningCode, clearLineCode
+
+      -- * Scrolling the screen
+    , scrollPageUpCode, scrollPageDownCode
+
+      -- * Select Graphic Rendition mode: colors and other whizzy stuff
+    , setSGRCode
+
+      -- * Cursor visibilty changes
+    , hideCursorCode, showCursorCode
+
+      -- * Changing the title
+    , setTitleCode
+    ) where
+
+import System.Console.ANSI.Types
+
+cursorUpCode, cursorDownCode, cursorForwardCode, cursorBackwardCode :: Int -- ^ Number of lines or characters to move
+                                                                    -> String
+cursorUpCode _       = ""
+cursorDownCode _     = ""
+cursorForwardCode _  = ""
+cursorBackwardCode _ = ""
+
+cursorDownLineCode, cursorUpLineCode :: Int -- ^ Number of lines to move
+                                     -> String
+cursorDownLineCode _ = ""
```

Comment:
What are all these `""` definitions useful for? Before, they were here so that the sets of variables defined on UNIX and Windows are the same. Now that you expose the proper codes to Windows, I don't think these are needed anymore.